src/components/Navbar/Navbar.jsx

```diff
@@ -1,16 +1,38 @@
-import { React } from 'react';
+import { useState, React } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { HStack, Flex, Button, Image, useMediaQuery, useToast } from '@chakra-ui/react';
+import {
+  HStack,
+  Flex,
+  Button,
+  Avatar,
+  Image,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Text,
+  useMediaQuery,
+  useToast,
+} from '@chakra-ui/react';
+import { FaSignOutAlt } from 'react-icons/fa';
 import NavbarLink from './NavbarLink';
 import ProfileDropdown from './ProfileDropdown';
 import logo from '../../assets/OCH_Logo_SVG.svg';
 import Toast from '../Toast';
 import NavbarMobile from './NavbarMobile';
 
-const Navbar = ({ isAdmin, changesMade }) => {
+const Navbar = ({ isAdmin, onAdminPortal, setOnAdminPortal, changesMade }) => {
+  const [adminPortal, setAdminPortal] = useState(true);
   const [isMobile] = useMediaQuery('(max-width: 1024px)');
 
+  console.log(typeof setOnAdminPortal);
+  const user = {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'user' is assigned a value but never used.